cilium-cli/connectivity/check/context.go

```diff
@@ -1211,3 +1227,26 @@ func (ct *ConnectivityTest) EchoServicePrefixes(ipFamily features.IPFamily) []ne
 	}
 	return res
 }
+
+// Multicast packet sender
+// This command exits with exit code 0
+// WITHOUT waiting for a second after receiving a packet.
+func (ct *ConnectivityTest) SocatServer1secCommand(peer TestPeer, port int, group string) []string {
+	addr := peer.Address(features.IPFamilyV4)
+	cmdStr := fmt.Sprintf("timeout 1 socat STDIO UDP4-RECVFROM:%d,ip-add-membership=%s:%s", port, group, addr)
```

Comment:
I think it may be good to make waiting-time longer. (Currently 1sec)
The reasons are
1. This command exits with exit-code 0 WITHOUT waiting for a second after receiving a packet.
2. If multicast configuration and implementation are correct, all pods in which this command is executed will receive multicast packet soon

This waiting-time will only be applied when there is an issue with the multicast implementation or configuration.

Reply:
A timeout does seem like the right failure condition for this test.

Do you have any concerns that there will be false negatives (failures), with a timeout of only 1 second?

I think 5 seconds should be fine here, if you have seen any such false negatives.

Reply:
I've changed.